app/src/modules/collections/routes/collection.vue

```diff
@@ -362,6 +363,8 @@ export default defineComponent({
 
 		const { batchEditAllowed, batchArchiveAllowed, batchDeleteAllowed, createAllowed } = usePermissions();
 
+		useShortcut('meta+a', gotoModalView);
```

Comment:
Lets remove the quick-switching to settings in favor of a right-click action on the left menu bar